src/hera.cpp

```diff
@@ -405,8 +405,15 @@ int hera_set_option(
   }
 
   if (strcmp(name, "engine") == 0) {
-    if (wasm_engine_options.count(value)) {
-      hera->wasm_engine = wasm_engine_options.at(value);
+    auto it = wasm_engine_options.find(value);
```

Comment:
I don't think the `wasm_engine_options` and `hera_wasm_engine` are needed any more. Here simple string switch will do.

Reply:
`string` switch? Is there such a thing in C++?